.github/workflows/build_docker_images.yml

```diff
@@ -72,6 +74,15 @@ jobs:
           push: true
           tags: atddocker/vz-socrata-export:${{ github.ref == 'refs/heads/production' && 'production' || 'development' }}
 
+      - name: Build and push VZ-Moped projects ETL image
+        if: steps.changes.outputs.moped_projects == 'true'
+        uses: docker/build-push-action@v4
+        with:
+          platforms: linux/amd64,linux/arm64
+          context: etl/moped_projects
+          push: true
+          tags: atddocker/vz-moped-join:${{ github.ref == 'refs/heads/production' && 'production' || 'development' }}
```

Comment:
hmm, yes, i don't think i've been very consistent with the dev/latest tagging across the board. if we were standardizing today, i think by preference would be to explicitly use the environment name, since that is a breadcrumb as to what's actually going on. but i do generally assume that `latest` means "bleeding edge build" 🤔